khronos_api/src/lib.rs

```diff
@@ -34,3 +34,6 @@ pub const WEBGL_IDL: &'static [u8] = include_bytes!("../api_webgl/specs/latest/1
 
 /// The contents of [`webgl2.idl`](https://github.com/KhronosGroup/WebGL/blob/master/specs/latest/2.0/webgl2.idl)
 pub const WEBGL2_IDL: &'static [u8] = include_bytes!("../api_webgl/specs/latest/2.0/webgl2.idl");
+
+/// The contents of the WebGL extension XML files
+pub const WEBGL_EXT_XML: &'static [&'static [u8]] = include!(concat!(env!("OUT_DIR"), "/webgl_exts.rs"));
```

Comment:
Any reason why you decided to go with this, instead of `&[(&str, &[u8])]`?

Reply:
I was originally going to extract the IDL in the `khronos_api` crate, but I decided that it would be better to keep the `khronos_api` crate as simple as possible, and just expose the raw XML. That way if I (or someone else) wants something else from the XML, then it's available. Given that I'm storing the whole XML, there's no need to store the name separately.